src/NuGet/BuildNuGets.csx

```diff
@@ -1,4 +1,4 @@
-#r "System.Xml.Linq"
+#r "System.Xml.XDocument.dll"
```

Comment:
This had me a little worried.  What's the best way for me to ensure this isn't going to break how this is used on Windows to produce nupkgs?  Do I get validation from CI?